ranger/ext/rifle.py

```diff
@@ -389,45 +394,39 @@ def execute(self, files,  # noqa: E501 pylint: disable=too-many-branches,too-man
                                          "Please set $TERMCMD manually or "
                                          "change fallbacks in rifle.conf.")
                         self._mimetype = 'ranger/x-terminal-emulator'
-                        self.execute(
-                            files=[command.split(';')[1].split('--')[0].strip()]
-                            + files, flags='f',
-                            mimetype='ranger/x-terminal-emulator')
+                        self.execute(files=cmd, flags='f',
+                                     mimetype='ranger/x-terminal-emulator')
                         return None
 
                     # Choose correct cmdflag accordingly
                     if term in ['xfce4-terminal', 'mate-terminal',
-                                'terminator']:
-                        cmdflag = '-x'
```

Comment:
Don't like the name, it's too generic. I want something that indicates these are the flags/arguments needed to start a command in the terminal.

Reply:
I avoided `cmdflag` because these are not actually flags _for_ `cmd`, but I agree.  I think `execflags` fits well.

Reply:
cmdflag as in the flag for executing a command. Execflags is fine with me though.